package.json

```diff
@@ -52,6 +52,7 @@
     "@typescript-eslint/parser": "^5.0.0",
     "audit-ci": "^5.0.0",
     "codecov": "^3.8.3",
+    "del": "^6.0.0",
```

Comment:
nit: should this be a `devDependency`?

Reply:
Already is 😄 
<img width="490" alt="Screen Shot 2021-11-19 at 11 32 34" src="https://user-images.githubusercontent.com/592134/142658064-8c8de6f2-ea1d-4bb9-8811-61f106983a1f.png">

Reply:
LOL does this mean my eyesight is going? Apologies!